### PR TITLE
Mturk data ranking repro

### PR DESCRIPTION
**Pull request by aatkinson**

### CODE REVIEW

File: competition/human_evaluation/human-evaluation-results.py [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+import click
+import logging
+import numpy as np
+import os
+import pandas as pd
+
+from collections import Counter, defaultdict
+from copy import deepcopy
+
+SEED = 2019
```
Comment (temporaer) [inline competition/human_evaluation/human-evaluation-results.py]: Maybe add a comment here that indicates that this seed was also used for data _collection_ and therefore cannot be changed?

File: competition/human_evaluation/human-evaluation-results.py [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+    cmps[row['Submission2']] += 1
+
+  normed_win_counts = Counter({k: win_counts[k] / cmps[k] for k in win_counts})
+
+  return win_counts, normed_win_counts
+
+
+@click.command()
+@click.argument('judgements_csv', type=click.Path(exists=True))
+@click.option('-d', '--rankings-dump-csv', default=None)
```
Comment (temporaer) [inline competition/human_evaluation/human-evaluation-results.py]: add help for this option

File: competition/human_evaluation/human-evaluation-results.py (hunk 1 above)
Comment (aatkinson) [reply competition/human_evaluation/human-evaluation-results.py]: good call, done

File: competition/human_evaluation/human-evaluation-results.py (hunk 2 above)
Comment (aatkinson) [reply competition/human_evaluation/human-evaluation-results.py]: done

**Pull request merged**